lib/TBDGen/TBDGen.cpp

```diff
@@ -299,6 +299,17 @@ void TBDGenVisitor::visitProtocolDecl(ProtocolDecl *PD) {
 #endif
 }
 
+void TBDGenVisitor::visitEnumDecl(EnumDecl *ED) {
+  if (!ED->isResilient(SwiftModule, ResilienceExpansion::Minimal))
```

Comment:
This is just `isResilient()`. The module you're supposed to pass is the client module, not the library module, but ResilienceExpansion::Minimal makes that uninteresting anyway.

Reply:
Like so: #15799